serve.go

```diff
@@ -177,6 +178,7 @@ func serve(args []string) error {
 	} else {
 		handler = r
 	}
+	handler = handlers.CombinedLoggingHandler(os.Stdout, handler)
```

Comment:
The other logger here goes to stderr
https://github.com/micromdm/micromdm/blob/472d416791f9ef822a6601763b0a6ffbc7c4b022/serve.go#L80

Thoughts on one or the other?